hawc/apps/lit/managers.py

```diff
@@ -559,6 +559,11 @@ def global_df(self) -> pd.DataFrame:
         ).values_list(*mapping.values())
         return pd.DataFrame(list(qs), columns=list(mapping.keys()))
 
+    def full_text_search(self, search_text):
```

Comment:
I think we need to convert to a SearchQuery too, so the words can be stemmed like the index: https://docs.djangoproject.com/en/4.2/ref/contrib/postgres/search/#searchquery.

Reply:
should we make the search_type="websearch" so that users can add AND/OR operators to the search?

Reply:
We also have the option to order by the relevance in the search filter. Would that be good to add in this card?

Reply:
we can try websearch, as long as the a simple "foo 2012" search still works like we'd expect.  I'd try to order by relevance too if you can; though i think the order by may interfere w/ the order by in our filterset